assets/javascripts/discourse/initializers/preview-edits.js.es6

```diff
@@ -160,6 +160,15 @@ export default {
           this.$('.topic-intro').prependTo(this.$('.main-link'))
           this.$('.topic-title').prependTo(this.$('.main-link'))
         }
+
+        if (socialMediaStyle) {
+          this.$().parents('#list-area').addClass('social-media')
+        } else {
+          this.$().parents('#list-area').removeClass('social-media')
+          if (Discourse.SiteSettings.topic_list_social_media_only_latest) {
+            this.$('.topic-thumbnail').remove()
```

Comment:
@pmusaraj Could you make this part a different setting? That will make this feature a little bit more usable by other folks as well. Also, please toggle the ``showThumbnail`` property rather than removing the element.